ramls/edge-oai-pmh.raml

```diff
@@ -7,7 +7,7 @@ documentation:
     title: 'Edge API - OAI-PMH'
     content: 'Edge API to interface with FOLIO for 3rd party harvesters to harvest metadta via OAI-PMH'
 types:
-  OAI-PMH: !include schemas/OAI-PMH.xsd
+#  OAI-PMH: !include schemas/OAI-PMH.xsd
```

Comment:
I can propose to remove this instead of using comment.

Reply:
done